src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/TextEditingTarget.java

```diff
@@ -2211,6 +2212,24 @@ void onReformatCode()
    }
    
    @Handler
+   void onRenameInFile()
+   {
+      int matches = renameHelper_.renameInFile();
```

Comment:
For C++ mode I think we might need to exclude the command entirely (so users don't think they can rename C++ variables).